pkg/recommender.py

```diff
@@ -425,3 +436,81 @@ def record_data(self, data):
 
     storing_db.close()
 
+  def _schedule_evt(self):
+    '''
+    Send the morning message at 10 am
+    '''
+    next_morning = ''
+    while True:
+      now = datetime.now()
+
+      #check if we have passed 10 am yet
+      if now.hour >= 10: #then you are currently in the day before
+        #add a day to the date
+        next_morning = now + timedelta(days =1)
+        #change the time to 10am
+        next_morning = next_morning.replace(hour = 10, minute = 0, second = 0, microsecond = 0)
+
+      elif now.hour < 10: #then we are at the same day
```

Comment:
this condition is unnecessary, we can change the code to be more concise
we will update the hour anyway

```python
if now.hour < 10:
  next_morning = now + timedelta(days =1)

next_morning = next_morning.replace(hour = 10, minute = 0, second = 0, microsecond = 0)
```